extmod/extmod.mk

```diff
@@ -1,5 +1,54 @@
 # This makefile fragment provides rules to build 3rd-party components for extmod modules
```

Comment:
This comment will need updating, because it now also adds all extmod source code.